.tributors

```diff
@@ -62,6 +62,13 @@
         "name": "Agah Karakuzu",
         "orcid": "0000-0001-7283-271X"
     },
+    "agnese_zazio": {
+        "contributions": [
+            "doc"
+        ],
+        "email": "agnese.zazio@cognitiveneuroscience.it",
+        "name": "Agnese Zazio"
```

Comment:
public info at https://www.cognitiveneuroscience.it/chi-siamo/?lang=en
and https://scholar.google.com/citations?user=3uyCWmMAAAAJ&hl=it

didn't find github handle